src/message.rs

```diff
@@ -214,7 +233,9 @@ mod test {
     use tokio::sync::RwLock;
     use twilight_model::id::Id;
 
+    use super::MentionTrait;
```

Comment:
prefer importing this from `twilight` itself; this makes it appear like `MentionTrait` is coming from Chrysanthemum itself
```suggestion
    use twilight_mention::Mention as MentionTrait;
```